### DISCUSSION TITLE
Unable to access the terminal of the work node on the UI page

### DISCUSSION THREAD

**Discussion by lizhaoyangre** [okd-project/okd#1989] [General]
My OKD cluster version is 4.14. After installation, I found that I couldn't access the worker node's terminal page on the UI page. What should I do?
I found that this problem is caused by the inability to pull the debug image. My cluster is on a restricted network. How can I modify this image source?
![image](https://github.com/user-attachments/assets/6892d50c-2b40-4c49-afa6-b3a965256009)

**Comment by melledouwsma** [comment]
Did you set up this cluster as a disconnected installation? If so, there should be an `ImageContentSourcePolicy` on your cluster that maps the original image location to your mirrored registry. This should allow `oc debug node` to start normally.

You can also customize the image using the `--image` option. For more information, please refer to [How to use a different image with toolbox and oc debug node in OCP 4](https://access.redhat.com/solutions/4929021).
